### ISSUE TITLE
TaxonWorks

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#2524]
Here is a platform for taxonomy that we need to evaluate.  
http://taxonworks.org/

**Comment by dustymc** [comment]
I think that's more of a CMS-lite than a taxonomy platform?

**Comment by Jegelewicz** [comment]
See #2176

**Comment by campmlc** [comment]
It will be the taxonomy source for The Parasite Tracker TCN, which involves
Arctos collections.

**Comment by Jegelewicz** [comment]
I've looked at their site and it is pretty much Greek to me, but maybe someone with both knowledge of taxaonomy and technology can figure it out. @mkoo ?

**Comment by Jegelewicz** [comment]
http://taxonworks.org/

**Comment by campmlc** [comment]
They are going to set up a sandbox for us to play around in.

**Comment by Jegelewicz** [comment]
@campmlc any news on this? Should we set up a meeting with Matt and Dusty?

**Comment by campmlc** [comment]
Yes, good idea, but I don't have the bandwidth to set it up just yet. I do
have all the current TPT taxonomy files that they are using from Symbiota
and Specify that we can import. They are also asking for us to send them
relevant Arctos taxonomy.

**Comment by dustymc** [comment]
This is a LONG way from where I need to be involved!

If they're doing something useful, you should encourage them to talk to GlobalNames (in which case the data will just appear in Arctos).

If that's somehow insufficient (there's no technical reason it would be), and you want to use these data for something like managing a collection, and they have a semi-sane API hiding somewhere, then we could treat them like WoRMS. (And I probably still don't need to talk to their developer...)

**Comment by Jegelewicz** [comment]
As far as I can tell, their API is not yet functional, which is why we need to talk to someone.

**Comment by dustymc** [comment]
> API is not yet functional

Then there's definitely no point in me talking to them.

I'm willing if ya'll find it useful, this does no look remotely like a technical problem. Once they can demonstrate something that someone will commit to using for managing a collection, then I can contribute. Before then, I'm not sure what I can bring to the table.

**Comment by Jegelewicz** [comment]
These guys were brought up today by DINA - https://github.com/ArctosDB/newsletter/issues/110 but I am still unable to find any helpful usage examples.

**Comment by dustymc** [comment]
> unable to find any helpful 

I think this has been sufficiently evaluated.
